lib/awful/widget/prompt.lua

```diff
@@ -57,7 +57,7 @@ end
 --
 -- @param args Arguments table. "prompt" is the prompt to use.
 -- @return A launcher widget.
--- @name awful.widget.prompt
+-- @function awful.prompt
```

Comment:
My mistake for the error, I should have found it in the review that introduced it and the CI didn't complain.

Daniel is right, it should be `-- @function awful.widget.prompt`